decidim-surveys/lib/decidim/surveys/user_responses_serializer.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Surveys
+    class UserResponsesSerializer < Decidim::Exporters::Serializer
+      include Decidim::TranslationsHelper
+
+      # Public: Exports a hash with the serialized data for the user response.
+      def serialize
+        # Returns a csv export of surveys responses only if they have been published.
+        response = resource
+        {
+          id: response.session_token,
+          created_at: response.created_at,
+          ip_hash: response.ip_hash,
+          user_status: (response.decidim_user_id.present? ? "Registered" : "Unregistered"),
+          question: question_text(response),
+          body: normalize_body(response)
+        }
+      end
+
+      private
+
+      def question_text(response)
+        if response.question.present?
+          "#{response.question.position}. #{translated_attribute(response.question.body)}"
+        else
+          ""
+        end
+      end
+
+      def normalize_body(response)
+        return response.body if response.body.present?
+
+        normalize_choices(response.choices)
+      end
+
+      def normalize_choices(choices)
+        choices.map { |choice| translated_attribute(choice.try(:body)) }.compact.join(", ")
+      end
+
+      def translated_attribute(attribute)
+        if attribute.is_a?(Hash)
+          attribute[I18n.locale.to_s] || attribute["en"] || attribute.values.first
```

Comment:
It doesn't make sense to have "en" here. 
I would actually try to access `organization.default_locale`